inference-engine/src/transformations/src/transformations/op_conversions/convert_mvn1_to_mvn6.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_mvn1_to_mvn6.hpp"
+
+#include <ngraph/rt_info.hpp>
+
+#include <numeric>
+
+#include <ngraph/opsets/opset2.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertMVN1ToMVN6, "ConvertMVN1ToMVN6", 0);
+
+ngraph::pass::ConvertMVN1ToMVN6::ConvertMVN1ToMVN6() {
+    auto mvn = pattern::wrap_type<ngraph::op::v0::MVN>();
+
+    ngraph::matcher_pass_callback callback = [](pattern::Matcher& m) {
+        auto match_root = m.get_match_root();
+        auto mvn_node = std::dynamic_pointer_cast<ngraph::op::v0::MVN>(match_root);
+        if (!mvn_node) {
+            return false;
+        }
+
+        const auto input = mvn_node->input_value(0);
+
+        // MVN-1 support only 4D and higher input tensors
+        auto input_rank = input.get_partial_shape().rank();
+        if (!input_rank.is_static() || input_rank.get_length() < 4) {
```

Comment:
Why do we check the rank length?
If MVN6 has no constraints for the input shape (and based on the [operations spec](https://github.com/openvinotoolkit/openvino/pull/3314/files#diff-b49d70cbefe387b7c810646a5c66145016d5821d3cff928ec0eeba21e93c8398R59) it doesn't) then every MVN1 can be converted to MVN6. So I don't see any reason to keep this check.

Reply:
MVN1 only supported 4D and 5D tensors

Reply:
Okay, but here you replace MVN1 with MVN6 so there is no need to check that original node (MVN1) has some particular shapes because it already has valid shape.